<?xml version="1.0" encoding="utf-8"?>

<launch>
  
  <!-- Load parameters -->
  <rosparam command="load" ns="imgs_to_rosvideo_node" file="$(find imgs_to_rosvideo)/param/default.yaml"/>

  <!-- Start darknet and ros wrapper -->
  <node pkg="imgs_to_rosvideo" type="imgs_to_rosvideo_node" name="imgs_to_rosvideo_node" output="screen">
  </node>


<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="imgs_to_rosvideo_node/image_raw" />
</node>

</launch>
